lib/lib/emojis.tsx

```diff
@@ -0,0 +1,53 @@
+import { cva, type VariantProps } from "class-variance-authority"
+import { parse } from "twemoji-parser"
+
+interface ParseObject {
+  url: string
+  indices: [number, number]
+  text: string
+}
+
```

Comment:
Small comment here, don't we have two levels of `lib` here? Maybe it's better to have something like `lib/emojis/index.tsx` instead of `lib/lib/emojis.tsx`

Reply:
Umm, it seems weird, but we have almost everything inside the first lib folder (components, experimental, ui...), so it doesn't seem right to have emojis at the same level, does it?